### ISSUE TITLE
Migrate Docker containers for French apps

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/ofn-install#326]
This is currently blocking #270.

There are 3 French servers containing:
- Production
- Staging and 2 Docker containers for the Wordpress and Discourse
- A spare server (old Production)

We need to migrate the 2 Docker apps from Staging to the spare server and make sure they are running ok.

This will enable us to set up French Staging properly with ofn-install.

**Comment by Matt-Yorkley** [comment]
@luisramos0 maybe you could help @pacodelaluna with this one if you have some time?

**Comment by luisramos0** [comment]
I don't have any practice with docker (I only know the theory very well).
I remember last year I spent almost a day just finding out how to restart these containers 🤣 
Having said that...

@pacodelaluna do you want to give me access to the spare server and some hints on how do I migrate these docker containers to it?

Preserve and move the docker images should be straight forward, but I have no idea how to move the data volumes.... I can [stack overflow it](https://stackoverflow.com/questions/21597463/how-to-port-data-only-volumes-from-one-host-to-another)...

**Comment by pacodelaluna** [comment]
@luisramos0 Please assign me the issue, I will work on it. Don't know how long it will take but at least I can share my view upon it.

**Comment by luisramos0** [comment]
ok, thanks. I cant assign you to the issue. must be some gh group you dont belong to...

**Comment by sauloperez** [comment]
Welcome to the Developers team @pacodelaluna ! I added you myself

**Comment by pacodelaluna** [comment]
Ok, then it seems I succeeded in moving the Discourse instance.
The process I followed is:
* Get a fresh install of Discourse on new server (https://github.com/discourse/discourse/blob/master/docs/INSTALL-cloud.md) with the existing configuration (/var/discourse/containers/app.yml)
* Migrate data and files to new instance (using the shared folder allowing to share easily files between local file system and Docker instance)
* Rebuild the Discourse app
* Import Nginx config and SSL certificates from old server
* Test from local machine using specific host entry
* Update DNS record on registar

Next step, to move the Wordpress instance, to be continued... ;)

**Comment by luisramos0** [comment]
nice, well done!!!

**Comment by pacodelaluna** [comment]
I have succeeded in migrating the wordpress instance from the old stage. Same strategy than for Discourse, fresh install from the imported `docker-compose.yml` file, import of files and data from tar archive and database dump, Nginx config, ...

Fyi, I have added keys of @luisramos0, @sauloperez and @mkllnk on the FR Services server (I am buying a better name for this server, no problem ;), we could add more people if required.

@RachL @myriamboure I have tested the forum and the blog, it looks good to me but I am letting you confirm that here, then we will be able reset completely the FR Stage server and have a brand new server for tests!
